modules/social_features/social_user/src/Plugin/better_exposed_filters/filter/Html5Date.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Drupal\social_user\Plugin\better_exposed_filters\filter;
+
+use Drupal\better_exposed_filters\Plugin\better_exposed_filters\filter\FilterWidgetBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * HTML5 date widget implementation.
```

Comment:
@VBaldych could you explain why this plugin is needed, where it's used and what you take as a base for it?

Reply:
The main idea to create this filter widget is avoid using **form_alter** with hardcode for Date filters. In our case, we use it on 'Registered' and 'Last active' fields